src/daemon/jsonrpc/api.rs

```diff
@@ -1075,9 +1076,49 @@ impl RpcApi for RpcImpl {
                     continue;
                 }
             }
+
+            let spent_vaults = db_vaults_from_spend(&db_path, &db_spend.psbt.txid())
+                .map_err(|e| internal_error!(e))?;
+            let derivation_index = spent_vaults
+                .values()
+                .map(|v| v.derivation_index)
+                .max()
+                .expect("Spent vaults should not be empty");
+            let cpfp_address = revaultd
+                .cpfp_descriptor
+                .derive(derivation_index, &revaultd.secp_ctx)
+                .into_inner()
+                .script_pubkey();
+            let deposit_address = revaultd
+                .deposit_descriptor
+                .derive(derivation_index, &revaultd.secp_ctx)
+                .into_inner()
+                .script_pubkey();
+            let mut cpfp_index = None;
+            let mut change_index = None;
+            for (i, txout) in db_spend
+                .psbt
+                .inner_tx()
+                .global
+                .unsigned_tx
+                .output
+                .iter()
+                .enumerate()
+            {
+                if cpfp_address == txout.script_pubkey {
+                    cpfp_index = Some(i);
+                }
+
```

Comment:
So currently (and this is unlikely to change anytime soon) the CPFP output is always going to be the first one and the change output the last one:
CPFP: https://github.com/revault/revault_tx/blob/6a0c2759332e4674e7a1f288241b0a546422d4f5/src/transactions.rs#L1203-L1208
Change: https://github.com/revault/revaultd/blob/062abff12e2dc3800b2bdf21b8853adae6a6df92/src/daemon/jsonrpc/api.rs#L952-L967

It's brittle but we should try to stick to it in the edgy edge case that they have multiple CPFP outputs. Therefore:
```rust
                if cpfp_address == txout.script_pubkey {
                    if cpfp_index.is_none() {
                        cpfp_index = Some(i);
                    }
                }
```